src/area-chart/model/use-chart-model.ts

```diff
@@ -319,11 +319,7 @@ export default function useChartModel<T extends AreaChartProps.DataTypes>({
     };
 
     const onPopoverLeave = (event: MouseEvent) => {
-      // pointing device exited from the page or from an element within the svg
-      if (
-        (event.relatedTarget !== window && plotRef.current!.svg.contains(event.relatedTarget as Node)) ||
-        interactions.get().isPopoverPinned
-      ) {
+      if (nodeContains(plotRef.current!.svg, event.relatedTarget as Element) || interactions.get().isPopoverPinned) {
```

Comment:
I think this type-casting is not needed. I specifically updated the utility signature to accept `relatedTarget`: https://github.com/cloudscape-design/component-toolkit/blob/fa9186592934151b720c12070239ec4ded7c34a9/src/dom/node-contains.ts#L9

But thanks for using this utility anyway, I can approve this now